content/cli/tokens.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: bt_wiki
+title: tokens
+category: Docs
+draft: false
+abstract: Cloudify's Command-Line Interface
+weight: 227
+---
+
+The `cfy tokens` command is used to receive a REST token from the Cloudify Manager.
+
+## Commands
+
+### get
+
+#### Usage
+`cfy tokens get [OPTIONS]`
+
+returns a valid REST token from the Cloudify Manager
```

Comment:
returns > Returns
Please add periods at the end of this sentence and at the end of "You can apply..." below.